lightning-rapid-gossip-sync/src/processing.rs

```diff
@@ -623,4 +624,57 @@ mod tests {
 			panic!("Unexpected update result: {:?}", update_result)
 		}
 	}
+
+	fn generate_blank_rgs_blob(chain_hash: &BlockHash, timestamp: u32) -> Vec<u8> {
```

Comment:
This is required because the serialisation code is in the [example server](https://github.com/lightningdevkit/rapid-gossip-sync-server/blob/main/src/serialization.rs#L92), right?